Permission tests are failing for the 1.0a version:
```
=========================== short test summary info ============================
FAILED tests/test_secrets.py::test_permissions[admin-/-/secrets-GET-None] - assert 403 != 403
 +  where 403 = <Response [403 Forbidden]>.status_code
FAILED tests/test_secrets.py::test_permissions[admin-/-/secrets/EXAMPLE_SECRET-GET-None] - assert 403 != 403
 +  where 403 = <Response [403 Forbidden]>.status_code
FAILED tests/test_secrets.py::test_set_secret - assert 403 == 302
 +  where 403 = <Response [403 Forbidden]>.status_code
FAILED tests/test_secrets.py::test_get_secret - assert 403 == 302
 +  where 403 = <Response [403 Forbidden]>.status_code
FAILED tests/test_secrets.py::test_secret_index_page - assert 403 == 200
 +  where 403 = <Response [403 Forbidden]>.status_code
=================== 5 failed, 6 passed, 20 warnings in 0.90s ===================
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,7 +25,7 @@ CI = "https://github.com/datasette/datasette-secrets/actions"
 secrets = "datasette_secrets"
 
 [project.optional-dependencies]
-test = ["pytest", "pytest-asyncio", "datasette-test>=0.3"]
+test = ["pytest", "pytest-asyncio", "datasette-test>=0.3.2"]
 
 [tool.pytest.ini_options]
 asyncio_mode = "strict"

